File: app/src/main/java/ru/vincetti/vimoney/transaction/debt/TransactionDebtFragment.java
package ru.vincetti.vimoney.transaction.debt;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import ru.vincetti.vimoney.R;
import ru.vincetti.vimoney.data.models.TransactionModel;
import ru.vincetti.vimoney.transaction.TransactionFragmentInterface;

public class TransactionDebtFragment extends Fragment {

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_add_spent, container, false);
    }

}

File: app/src/main/java/ru/vincetti/vimoney/ui/transaction/debt/TransactionDebtFragment.kt
package ru.vincetti.vimoney.ui.transaction.debt

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import ru.vincetti.vimoney.databinding.FragmentAddSpentBinding

class TransactionDebtFragment : Fragment() {

    private var _binding: FragmentAddSpentBinding? = null
    private val binding
        get() = requireNotNull(_binding)

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        _binding = FragmentAddSpentBinding.inflate(layoutInflater)
        return binding.root
    }

    override fun onDestroyView() {
        super.onDestroyView()
        _binding = null
    }
}
